core/modules/savers/beaker.js

```diff
@@ -51,7 +51,7 @@ BeakerSaver.prototype.info = {
 Static method that returns true if this saver is capable of working
 */
 exports.canSave = function(wiki) {
-	return !!window.DatArchive;
+	return !!window.DatArchive && location.protocol==="dat:";
```

Comment:
yea, but the first condition will fail in any browser except the beaker browser. IMO this is a feature test. If the API is available it should be able to use it. 

And beaker browser is in flux. ... There was a "workspace:" protocol that also made dat API available. .. So this check would have been a "false", which would have been an issue.

Reply:
I don't think, that this should be merged.